### DISCUSSION TITLE
Welcome to NCI Dose Forum Discussions!

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#20] [Announcements]
## 👋 Welcome to the NCI Dose Forum!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Request new features.
- Welcome others and be open-minded. Remember that this is a community we build together.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ramiromorenopro** [comment]
Hi, thank you for inviting me, I notice a typo in the welcome text .. "peditric" as pediatric..
Ramiro

**Reply by ramiromorenopro** [reply]
in the readme more precisely..

**Reply by choonsiklee** [reply]
Fixed. Thanks!

**Comment by andreasjahnen** [comment]
Dear All, my name is Andreas Jahnen from Luxembourg. I work at the Luxembourg Institute of Science and Technology. My research interests are radiation protection (and IT tools connected with that), radiation epidemiology and AI in Health.

Looking forward to exchange in this forum!

Andreas

**Reply by choonsiklee** [reply]
Welcome Andreas!
